kitsune/notifications/tests/test_api.py

```diff
@@ -38,3 +47,68 @@ def test_cant_register_for_other_users(self):
         eq_(serializer.errors, {
             'creator': ["Can't register push notifications for another user."],
         })
+
+
+class TestNotificationSerializer(TestCase):
+    def test_correct_fields(self):
+        follower = profile()
+        followed = profile()
+        q = question(creator=followed.user, save=True)
+        follow(follower.user, followed.user)
```

Comment:
I don't understand this. Is the notification below getting sent to the follower? Is this just a test case or is sumo becoming twitter with followers and all that?